integration/ibc/cmd_relayer_test.go

```diff
@@ -0,0 +1,633 @@
+		)),
+	))
+
+	env.Must(env.Exec("create a sentPost type list in an IBC module",
+		step.NewSteps(step.New(
+			step.Exec(envtest.IgniteApp,
+				"s",
+				"list",
+				"sentPost",
+				"postID:uint",
+				"title",
+				"chain",
+				"--no-message",
+				"--module",
+				"blog",
+				"--yes",
+			),
+			step.Workdir(app.SourcePath()),
+		)),
+	))
+
+	env.Must(env.Exec("create a timeoutPost type list in an IBC module",
+		step.NewSteps(step.New(
+			step.Exec(envtest.IgniteApp,
+				"s",
+				"list",
+				"timeoutPost",
+				"title",
+				"chain",
+				"--no-message",
+				"--module",
+				"blog",
+				"--yes",
+			),
+			step.Workdir(app.SourcePath()),
+		)),
+	))
+
+	env.Must(env.Exec("create a ibcPost package in an IBC module",
+		step.NewSteps(step.New(
+			step.Exec(envtest.IgniteApp,
+				"s",
+				"packet",
+				"ibcPost",
+				"title",
+				"content",
+				"--ack",
+				"postID:uint",
+				"--module",
+				"blog",
+				"--yes",
+			),
+			step.Workdir(app.SourcePath()),
+		)),
+	))
+
+	blogKeeperPath := filepath.Join(app.SourcePath(), "x/blog/keeper")
+	require.NoError(t, goanalysis.ReplaceCode(
+		blogKeeperPath,
+		nameSendIbcPost,
+		funcSendIbcPost,
+	))
+	require.NoError(t, goanalysis.ReplaceCode(
+		blogKeeperPath,
+		nameOnRecvIbcPostPacket,
+		funcOnRecvIbcPostPacket,
+	))
+	require.NoError(t, goanalysis.ReplaceCode(
+		blogKeeperPath,
+		nameOnAcknowledgementIbcPostPacket,
+		funcOnAcknowledgementIbcPostPacket,
+	))
+	require.NoError(t, goanalysis.ReplaceCode(
+		blogKeeperPath,
+		nameOnTimeoutIbcPostPacket,
+		funcOnTimeoutIbcPostPacket,
+	))
+
+	// serve both chains.
+	ports, err := availableport.Find(
+		14,
+		availableport.WithMinPort(4000),
+		availableport.WithMaxPort(5000),
+	)
+	require.NoError(t, err)
+	earthAPI, earthRPC, earthGRPC, earthFaucet := runChain(t, ctx, env, app, earthConfig, tmpDir, ports[:7])
+	earthChainID := earthConfig.Genesis["chain_id"].(string)
+	earthHome := earthConfig.Validators[0].Home
+	marsAPI, marsRPC, marsGRPC, marsFaucet := runChain(t, ctx, env, app, marsConfig, tmpDir, ports[7:])
+	marsChainID := marsConfig.Genesis["chain_id"].(string)
+	marsHome := marsConfig.Validators[0].Home
+
+	// check the chains is up
+	stepsCheckChains := step.NewSteps(
+		step.New(
+			step.Exec(
+				app.Binary(),
+				"config",
+				"output", "json",
+			),
+			step.PreExec(func() error {
+				if err := env.IsAppServed(ctx, earthAPI); err != nil {
+					return err
+				}
+				return env.IsAppServed(ctx, marsAPI)
+			}),
+		),
+	)
+	env.Exec("waiting the chain is up", stepsCheckChains, envtest.ExecRetry())
+
+	// ibc relayer.
+	env.Must(env.Exec("install the hermes relayer app",
+		step.NewSteps(step.New(
+			step.Exec(envtest.IgniteApp,
+				"app",
+				"install",
+				"-g",
+				// filepath.Join(goenv.GoPath(), "src/github.com/ignite/apps/hermes"), // Local path for test proposals
+				"github.com/ignite/apps/hermes",
+			),
+		)),
+	))
+
+	env.Must(env.Exec("configure the hermes relayer app",
```

Comment:
OK, this does create a default channel for `transfer` port which later gets used to test `ibc-transfer` tx but i don't see a dedicated port and channel for `send-ibc-post` tx (`blog' ibc module). How this unit test is going to test `blog` ibc module?

Reply:
the other tests will be implemented in another PR. The current test already covers the problem of testing IBC transfers. But now we already have everything to test custom transfers too. We only need to create the transfers